### PR TITLE
0.1.18 iready overload

### PR DESCRIPTION
**Pull request by FuckIready123**
Bro cupidity can you make a bookmark version of the new 0.1.18 version of the iready overload because I can't use the extension version.

### CODE REVIEW

**Comment by Auzgame** [comment]
@cupiditys There at it again lol.
`function genTitle() {
    let _0x57fca0 = ''
      , _0xe3361 = '឵\u200a';
    _0x57fca0 = _0x57fca0 + _0xe3361,
    _0x57fca0 = _0x57fca0 + 'i',
    _0x57fca0 = _0x57fca0 + _0xe3361,
    _0x57fca0 = _0x57fca0 + 'Reа',
    _0x57fca0 = _0x57fca0 + 'dy',
    _0x57fca0 = _0x57fca0 + '‌‌&nbsp;OverIoаd&nbsp;';
    for (let _0x52e996 = -0x1353 + 0xedb + 0x478; _0x52e996 <= Math['floor'](Math['random']() * Math['floor'](Math['random']() * (-0x161 * 0x17 + -0x16 * 0xef + 0x344d))); _0x52e996++) {
        _0x57fca0 = _0x57fca0 + _0xe3361;
    }
    return _0x57fca0;
}`

**Comment by oTrentk** [comment]
What was that for? Is that the code

**Comment by LiLD3MON** [comment]
can you make the javascript code for the bookmark version developer mode cant be turned on due to school blocks

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 <h1 align="center">iReady Overload</h1>
 
+		note: I will no longer be releasing updates on github, only in the discord server below (click the icon)
```
Comment (LiLD3MON) [inline README.md:5]: i cant open discord on my school computer can you put the bookmark code on github

**Comment by wtfstretch** [comment]
farm mins dont work

File: README.md (hunk 1 above)
Comment (pusscee) [reply README.md:5]: Go to google or microsoft edge and type in vpn extensions, and find the one where it says hoxx vpn. It basically unblocks everything

Comment (Auzgame) [reply README.md:5]: @pusscee  thats not necessarily true. As of right now my school blocks extensions.

Comment (Gam3rrXD) [reply README.md:5]: Use a download vpn

**Pull request closed without merging**